addons/pos_epson_printer/static/src/js/printers.js

```diff
@@ -2,98 +2,167 @@
 odoo.define('pos_epson_printer.Printer', function (require) {
 "use strict";
 
-const { Gui } = require('point_of_sale.Gui');
 var core = require('web.core');
-var PrinterMixin = require('point_of_sale.Printer').PrinterMixin;
+var { PrinterMixin, PrintResult, PrintResultGenerator } = require('point_of_sale.Printer');
 
+var QWeb = core.qweb;
 var _t = core._t;
 
+class EpsonPrintResultGenerator extends PrintResultGenerator {
+    constructor(address) {
+        super();
+        this.address = address;
+    }
+
+    IoTActionError() {
+        var printRes = new PrintResult({
+            successful: false,
+            message: {
+                title: _t('Connection to the printer failed'),
+                body: _t('Please check if the printer is still connected.'),
+            }
+        });
+
+        if (window.location.protocol === 'https:') {
+            printRes.message.body += _.str.sprintf(_t(`
```

Comment:
don't use ` for multi-line translated content, we will probably likely ban it soon. In this case, there is unnecessary blank spaces.
https://discord.com/channels/678381219515465750/704997651833159831/813308760146575370